getblocklab/block-lab
# Use the jsx-a11y plugin to lint accessibility issues

#### Changes
* Adds an ESLint plugin to lint for accesiblity, on the recommendation of an [egghead.io course](https://egghead.io/courses/develop-accessible-web-apps-with-react)
* Corrects an issue this found 
#### Testing instructions

Regression testing for the `<FetchInput>` component, used in the Post, Taxonomy, and User controls

1. Create a post 
2. Add a BL block that uses a Post, Taxonomy, or User control
3. Populate the control, and ensure it saves
